.github/workflows/cypress/action.yml

```diff
@@ -26,13 +26,14 @@ inputs:
 runs:
   using: 'composite'
   steps:
-    - uses: browser-actions/setup-chrome@v1
-
     - uses: ./.github/workflows/yarn
 
-    - name: Install Cypress
+    - name: Install Specific Chrome Version
       shell: bash
-      run: ./node_modules/.bin/cypress install
+      run: |
+        sudo apt-get install -y wget
+        sudo wget -q https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
```

Comment:
But stable current is also not a specific version

Reply:
I would stick to latest Chrome version so the tests run on latest version